frontend/src/bundles/auth/store/actions.ts

```diff
@@ -5,17 +5,20 @@ import {
     type UserSignUpRequestDto,
     type UserSignUpResponseDto,
 } from '~/bundles/users/users.js';
+import { StorageKey } from '~/framework/storage/storage.js';
 
 import { name as sliceName } from './slice.js';
 
 const signUp = createAsyncThunk<
     UserSignUpResponseDto,
     UserSignUpRequestDto,
     AsyncThunkConfig
->(`${sliceName}/sign-up`, (registerPayload, { extra }) => {
+>(`${sliceName}/sign-up`, async (registerPayload, { extra }) => {
     const { authApi } = extra;
```

Comment:
In this case you need to use storage from '~/framework/storage/storage.js' and throw it like authApi to extra args

```suggestion
    const { authApi, storage } = extra;
```